api/services/auth_test.go

```diff
@@ -196,10 +196,13 @@ func TestAuthUser(t *testing.T) {
 	}
 
 	for _, test := range tests {
```

Comment:
Instead of using an alias use:
```go
for _, tc := range tests {
```

Reply:
This is required by linter

https://github.com/shellhub-io/shellhub/actions/runs/2059037104